src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/HttpApi/ContentTypes.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.HttpApi
+{
+    internal static class ContentTypes
+    {
+        public const string ApplicationJson = "application/json";
```

Comment:
nit: I'm wondering if these are defined somewhere in the framework already?

Reply:
I've checked a handful of code and I don't think there's anything publicly available.

Reply:
`System.Net.Mime.MediaTypesName.Application.Json`